Eastern-Research-Group/mywaterway
# Feature/waterbody report accessibility

## Related Issues:
* [https://app.breeze.pm/projects/100762/cards/3333624](https://app.breeze.pm/projects/100762/cards/3333624)

## Main Changes:
* Changed some of the status badge text color to black.

Lighthouse and axe both flagged contrast errors on the white text on purple and white text on red backgrounds (see screenshots below). I used this [WebAIM Contrast Checker tool](https://webaim.org/resources/contrastchecker/) to find a foreground color with a higher contrast ratio, which ended up being black. I didn't want to mess with the background color as that is consistent with the status colors we are using everywhere else in the app. 

I prefer the way we have it now, but I figured I would create this pull request to get everyone's thoughts. 

Current
![image](https://user-images.githubusercontent.com/46329268/81970656-df0a3f80-95ed-11ea-8355-16fe3b7880f2.png)
![image](https://user-images.githubusercontent.com/46329268/81970693-edf0f200-95ed-11ea-9506-2eca00d214b8.png)

New
![image](https://user-images.githubusercontent.com/46329268/81972254-5345e280-95f0-11ea-9308-56c9c8eca900.png)
![image](https://user-images.githubusercontent.com/46329268/81972278-5a6cf080-95f0-11ea-8030-f7e4cf773461.png)


## Steps To Test:
1. Navigate to [http://localhost:3000/waterbody-report/MDE_EASP/MD-02140204](http://localhost:3000/waterbody-report/MDE_EASP/MD-02140204)
2. Run wave and check issues
3. Run axe and check issues
4. Run lighthouse and check issues
5. Navigate to [http://localhost:3000/waterbody-report/21AWIC/AL03150110-0202-200](http://localhost:3000/waterbody-report/21AWIC/AL03150110-0202-200)
6. Re-do steps 2 - 4.
